tests/smoke/auto-format-tools.sh

```diff
@@ -2,5 +2,4 @@
 
 SMOKE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-. ${SMOKE_DIR}/base.sh "use_black=yes use_blue=no"
-. ${SMOKE_DIR}/base.sh "use_black=no use_blue=yes"
+. ${SMOKE_DIR}/base.sh "use_black=no"
```

Comment:
We should test both the cases
```suggestion
. ${SMOKE_DIR}/base.sh "use_black=no"
. ${SMOKE_DIR}/base.sh "use_black=yes"
```

Reply:
Fixed it